tests/thread_flood/main.c

```diff
@@ -39,7 +39,15 @@ int main(void)
 {
     kernel_pid_t thr_id = KERNEL_PID_UNDEF;
     unsigned thr_cnt = 0;
+    unsigned thr_in_use = 0;
 
+    for (kernel_pid_t ix = KERNEL_PID_FIRST; ix <= KERNEL_PID_LAST; ix++) {
+        volatile thread_t *thread;
+        thread = thread_get(ix);
+        if (thread) {
```

Comment:
It also works with the following:
```suggestion
        if (thread_get(ix)) {
```
Is `volatile` really needed and, if yes, why ?

Reply:
Ah the volatile. I would be happy if these get removed, as in PR #11073
But it seems nobody dares to touch it.

Anyway, I've made a change to remove that variable

Reply:
I've cleaned up this initial loop because there is a `sched_num_threads` which already gives what we want.